app-backend/database/src/main/scala/com/azavea/rf/database/tables/Buckets.scala

```diff
@@ -223,6 +242,30 @@ object Buckets extends TableQuery(tag => new Buckets(tag)) with LazyLogging {
     }
   }
 
+  /** Adds a list of scenes to a bucket
+    *
+    * @param sceneIds Seq[UUID] list of primary keys of scens to add to bucket
+    * @param bucketId UUID primary key of back to add scenes to
+    */
+  def addScenesToBucket(sceneIds: Seq[UUID], bucketId: UUID)
+    (implicit database: DB): Future[Iterable[Scene.WithRelated]] = {
+
+    database.db.run {
+      ScenesToBuckets
+        .filter(sb => sb.bucketId === bucketId && sb.sceneId.inSet(sceneIds))
+        .map(_.sceneId)
+        .result
```

Comment:
The SQL in my head

```sql
SELECT sceneId FROM scenes_to_buckets WHERE bucketId = ??? AND sceneId IN (???)
```

seemed easier to translate to this `.filter().map().result` rather than a `for { } yield ()` comprehension, which I still don't have a good instinct for. If we should like to replace this with an equivalent `for` comprehension for the sake of consistency, I'm open to that.